MarkupsToModel/Logic/vtkSlicerMarkupsToModelLogic.cxx

```diff
@@ -266,24 +273,21 @@ void vtkSlicerMarkupsToModelLogic::UpdateSelectionNode( vtkMRMLMarkupsToModelNod
 //------------------------------------------------------------------------------
 void vtkSlicerMarkupsToModelLogic::UpdateOutputCloseSurfaceModel(vtkMRMLMarkupsToModelNode* markupsToModelModuleNode)
 {
-  vtkMRMLMarkupsFiducialNode* markups = markupsToModelModuleNode->GetMarkupsNode(); 
+  vtkMRMLMarkupsFiducialNode* markups = markupsToModelModuleNode->GetMarkupsNode();
   if(markups == NULL)
   {
-    vtkWarningMacro("No markups yet");
```

Comment:
Just describe when markup==NULL happens, why it's normal, and why it's OK to just return when it happens. This makes it clear for future developers why this decision has been made and it also forces you to think about it a bit if it's really the correct behavior to just return.

For example: if the user has a model generated from markups and then changes the MarkupsToModel node so that the markups node is NULL then we have to make sure that the model node is cleared (doesn't have any points or cells).